php/script.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+TARGET_DIR="pb-php"
+
+pf=(`find . -maxdepth 1 -name "*.proto"`)
+if [ ${#pf[@]} -eq 0 ]; then
+  echo "No proto files found!"
+  exit 1
+fi
+
+echo "Found Proto definitions:"
+printf "\t+%s\n" "${pf[@]}"
+
+echo 
+
+if [ ! -d "$TARGET_DIR" ]; then
+  mkdir $TARGET_DIR
+fi
+
+echo "Building Go source..."
```

Comment:
Hey @EwanValentine - So sorry to get to this so late, I'd love to merge this in, but would you mind updating the reference here to `go` please?